CHANGES.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+## 2.18.2
+
+- Support passing reporterOptions to mocha reporter via API [issue #144][]
```

Comment:
Thanks for taking care of the release notes also. Can I ask you to add the link to the issue (like below) and replace the `## 2.18.2` with something like `## Next`? I'd rather bump the version number when I actually do the release, because more than this one change might go in. Also, this is an API enhancement and is therefore going to be a minor 😉